From 69495314ff7a67b2957404169b75d1b2bd8610e4 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Fri, 4 Nov 2016 21:31:37 +0300
Subject: [PATCH 14/24] ARM64: dts: renesas: Add r8a77xx-panel.dtsi

Add r8a77xx-panel.dtsi that can be included in the r8a77xx device
trees for LVDS panel configuration. This will make it easier
to adjust and share panel configuration across Renesas boards.
Use 1024x768@60Hz panel mode.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi | 32 ++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)
 create mode 100644 arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi

diff --git a/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi b/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi
new file mode 100644
index 0000000..c7497bb
--- /dev/null
+++ b/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi
@@ -0,0 +1,32 @@
+/*
+ * Common file for the LVDS panel connected to Renesas R-Car boards
+ *
+ * Copyright (C) 2016 Cogent Embedded, Inc.
+ * Copyright (C) 2016 Renesas Electronics Corp.
+ *
+ * This file is licensed under the terms of the GNU General Public License
+ * version 2.  This program is licensed "as is" without any warranty of any
+ * kind, whether express or implied.
+ */
+
+/ {
+	lvds: lvds {
+		compatible = "lvds-connector";
+
+		width-mm = <210>;
+		height-mm = <158>;
+
+		panel-timing {
+			/* 1024x768 @60Hz */
+			clock-frequency = <65000000>;
+			hactive = <1024>;
+			vactive = <768>;
+			hsync-len = <136>;
+			hfront-porch = <20>;
+			hback-porch = <160>;
+			vsync-len = <6>;
+			vfront-porch = <3>;
+			vback-porch = <29>;
+		};
+	};
+};
-- 
1.9.3

